Dockerfile.dapper

```diff
@@ -50,16 +50,13 @@ ENV HELM_URL_V2_amd64=https://github.com/rancher/helm/releases/download/${CATTLE
     ETCD_URL_arm64=https://github.com/etcd-io/etcd/releases/download/${CATTLE_ETCD_VERSION}/etcd-${CATTLE_ETCD_VERSION}-linux-arm64.tar.gz \
     ETCD_URL=ETCD_URL_${ARCH} \
     KUSTOMIZE_URL_amd64=https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_amd64.tar.gz \
-    KUSTOMIZE_URL_arm64=https://github.com/brendarearden/kustomize/releases/download/kustomize/${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_arm64.tar.gz \
+    KUSTOMIZE_URL_arm64=https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_arm64.tar.gz \
```

Comment:
@brendarearden Is this safe? I don't recall why we had a branch

Reply:
they previously did not support an arm build so I built an arm version and pushed it to my own repo without any change to the upstream code. this is safe to do and makes sense now that they support multiple archs - 

Do we need both env vars now or can we do this:
 ```KUSTOMIZE_URL_${ARCH}=https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_${ARCH}.tar.gz```

Reply:
I think this and one other variables were configured based on previous ones were the URL was not generic to use variables in the value. I changed it for ETCD_URL and KUSTOMIZE_URL